### PR TITLE
add labels for all cases which has no labels

### PR DESCRIPTION
**Pull request by tingtli**
### The PR is to task341 add labels for cases without labels#

Here is the output of the labels check

[labelinfo.txt](https://github.com/xcat2/xcat-core/files/2583515/labelinfo.txt)

### CODE REVIEW

**Comment by hu-weihua** [comment]
Label ``hctrl_openbmc`` is not suitable for below cases:
```
clear_openbmc_simulator_multiple_100 others,hctrl_openbmc,
clear_openbmc_simulator_multiple_1000 others,hctrl_openbmc,
clear_openbmc_simulator_multiple_5000 others,hctrl_openbmc,
```

Is Label ``confignics`` in our label scope? It is a command name.
```
confignics_config_multiple_port_withnicaliases_multiple_value others,network,confignics,updatenode,
confignics_config_multiple_port_withnichostnamesuffixes_multiple_value others,network,confignics,updatenode,
confignics_config_multiple_ports_multiple_value others,network,confignics,updatenode,
confignics_config_multiple_ports_single_value others,network,confignics,updatenode,
confignics_config_one_port_single_value others,network,confignics,updatenode,
confignics_disable_set_to_1 others,network,confignics,updatenode,
confignics_disable_set_to_yes others,network,confignics,updatenode,
confignics_s others,network,confignics,updatenode,
```

Is Label ``conserver``in our label scope?


There is no labels for test cases about ``copycds`` ``nodeset_check`` ``pdu``.

There is no ``Horizontal Dimension Labels`` for below cases
```
encrypted_passwd  
genimage
makeknownhosts
mkdef_pdu_object 

```

Maybe we need update the value of ``os`` for install_xCAT_on_ubuntu
```
install_xCAT_on_rhels_sles xcat_install,os=Linux,
install_xCAT_on_ubuntu xcat_install,os=Linux,

```

Are below labels correct?
```
mkvm_i_l others,hctrl_hmc,hcp=hmc,
```


Some test cases about packimage invoking provision but without ``invoke_provision`` label.such like ``packimage_m_tar_c_gzip``.

**Comment by tingtli** [comment]
@hu-weihua what is the mistake for the pasting. i just copy/past the output of the command.

**Comment by hu-weihua** [comment]
@tingtli, there are multiple lines ``label:others,kdump,invoke_provision`` in test case ``linux_diskless_kdump``, Is it a typo?

**Comment by hu-weihua** [comment]
@tingtli Below are the difference between the cases with "ci_test" label and the cases in ``MN_basic.bundle``. It seems there is gap between them two even do not take commented out lines into account.   
```
[root@c910f03c17k25 tmp]# diff -y  ci_test  MN_basic.bundle
							      >	#check_mkdef_node_with_template_priority
							      >	#makedhcp_a_d_linux
							      >	#makedhcp_a_linux
							      >	#makedhcp_d_linux
							      >	#makedns
							      >	#makedns_d_node
							      >	#makedns_n
							      >	#makedns_node
							      >	#makehost_n_r
							      >	#makehosts_d
							      >	#makehosts_l
							      >	#makenetworks
							      >	#makenetworks_d
							      >	#mkdef_template_cec_template_step_by_step
							      >	#nodech_d_error
							      >	#xcatd_restart
							      >	#xcatsnap_v
assign_certain_command_permission				assign_certain_command_permission
chdef_dynamic_group						chdef_dynamic_group
chdef_group							chdef_group
chdef_group_p							chdef_group_p
chdef_m								chdef_m
chdef_multiple_keys						chdef_multiple_keys
chdef_n								chdef_n
chdef_null							chdef_null
chdef_p								chdef_p
chdef_template						      <
chdef_t_network							chdef_t_network
chdef_t_node							chdef_t_node
chdef_t_o_error							chdef_t_o_error
							      >	chdef_template
chdef_z								chdef_z
check_mn_cluster_log_file					check_mn_cluster_log_file
chtab_d								chtab_d
chtab_err_symble						chtab_err_symble
chtab_err_table							chtab_err_table
chtab_h								chtab_h
chtab_modify_key						chtab_modify_key
chtab_modify_node						chtab_modify_node
chtab_null							chtab_null
chtab_v								chtab_v
disable_root_permission_in_policy_table				disable_root_permission_in_policy_table
dumpxCATdb_a_p_nullskiptables					dumpxCATdb_a_p_nullskiptables
dumpxCATdb_a_p_nullskiptables_V					dumpxCATdb_a_p_nullskiptables_V
dumpxCATdb_a_p_skiptables					dumpxCATdb_a_p_skiptables
dumpxCATdb_h							dumpxCATdb_h
							      >	dumpxCATdb_p_V
dumpxCATdb_p_nullskiptables					dumpxCATdb_p_nullskiptables
dumpxCATdb_p_nullskiptables_V					dumpxCATdb_p_nullskiptables_V
dumpxCATdb_p_skiptables						dumpxCATdb_p_skiptables
dumpxCATdb_p_V						      <
dumpxCATdb_v							dumpxCATdb_v
							      >	gettab_H
gettab_err							gettab_err
gettab_h							gettab_h
gettab_H						      <
gettab_key_table						gettab_key_table
lsdef_a								lsdef_a
lsdef_nics							lsdef_nics
lsdef_null							lsdef_null
lsdef_s								lsdef_s
lsdef_t								lsdef_t
lsdef_t_auditlog						lsdef_t_auditlog
lsdef_template						      <
lsdef_template_switch_template				      <
lsdef_template_with_invalid_name			      <
lsdef_t_err							lsdef_t_err
lsdef_t_eventlog						lsdef_t_eventlog
lsdef_t_h_i							lsdef_t_h_i
lsdef_t_i_o							lsdef_t_i_o
lsdef_t_o_l							lsdef_t_o_l
lsdef_t_o_l_z							lsdef_t_o_l_z
lsdef_t_policy							lsdef_t_policy
lsdef_t_site							lsdef_t_site
lsdef_t_w							lsdef_t_w
							      >	lsdef_template
							      >	lsdef_template_switch_template
							      >	lsdef_template_with_invalid_name
lsxcatd_a							lsxcatd_a
lsxcatd_d							lsxcatd_d
lsxcatd_h							lsxcatd_h
lsxcatd_null							lsxcatd_null
							      >	makeconservercf_d
							      >	makeconservercf_noderange
							      >	makeconservercf_null
makedhcp_h							makedhcp_h
makedhcp_help							makedhcp_help
makedhcp_n							makedhcp_n
makedhcp_n_linux						makedhcp_n_linux
makedns_h							makedns_h
makehosts_h							makehosts_h
makehosts_help							makehosts_help
makehosts_n							makehosts_n
makehosts_n_noderange						makehosts_n_noderange
makehosts_null							makehosts_null
							      >	makeknownhosts_h
							      >	makenetworks_d_V
							      >	makenetworks_h
							      >	makenetworks_help
							      >	makenetworks_v
mkdef_dynamic_group						mkdef_dynamic_group
mkdef_f								mkdef_f
mkdef_github_issue2582						mkdef_github_issue2582
mkdef_group							mkdef_group
mkdef_multiple_keys						mkdef_multiple_keys
mkdef_node							mkdef_node
mkdef_node_with_a_node_template					mkdef_node_with_a_node_template
mkdef_null							mkdef_null
mkdef_regex_nicsip						mkdef_regex_nicsip
							      >	mkdef_t_network
							      >	mkdef_t_o_error
mkdef_template_cec_template_without_remainder			mkdef_template_cec_template_without_remainder
mkdef_template_invalid_template					mkdef_template_invalid_template
mkdef_template_switch_template_without_attribute		mkdef_template_switch_template_without_attribute
mkdef_t_network						      <
mkdef_t_o_error						      <
mkdef_z								mkdef_z
nodeadd_err_symbol						nodeadd_err_symbol
nodeadd_h							nodeadd_h
nodeadd_noderange						nodeadd_noderange
nodeadd_noderange_nodetype					nodeadd_noderange_nodetype
nodeadd_null							nodeadd_null
nodeadd_v							nodeadd_v
nodech_delete							nodech_delete
nodech_error_node						nodech_error_node
nodech_error_table						nodech_error_table
nodech_h							nodech_h
nodech_noderanage_table_at					nodech_noderanage_table_at
nodech_noderange_shortname_groups				nodech_noderange_shortname_groups
nodech_noderange_shortname_mgt					nodech_noderange_shortname_mgt
nodech_noderange_shortname_tags					nodech_noderange_shortname_tags
nodech_noderange_table						nodech_noderange_table
nodech_noderange_table_arrow					nodech_noderange_table_arrow
nodech_noderange_table_comma					nodech_noderange_table_comma
nodech_noderange_table_comma_arrow				nodech_noderange_table_comma_arrow
nodech_noderange_table_equal					nodech_noderange_table_equal
nodech_noderange_table_include					nodech_noderange_table_include
nodech_noderange_table_unequal					nodech_noderange_table_unequal
nodech_noderange_table_uninclude				nodech_noderange_table_uninclude
nodech_v							nodech_v
nodegrpch_err							nodegrpch_err
nodegrpch_groups						nodegrpch_groups
nodegrpch_h							nodegrpch_h
nodegrpch_v							nodegrpch_v
							      >	nodels_H
							      >	nodels_S
nodels_b							nodels_b
nodels_err_noderange						nodels_err_noderange
nodels_err_symbol						nodels_err_symbol
nodels_h							nodels_h
nodels_H						      <
nodels_noderange						nodels_noderange
nodels_noderange_shortname_groups				nodels_noderange_shortname_groups
nodels_noderange_shortname_mgt					nodels_noderange_shortname_mgt
nodels_noderange_shortname_tags					nodels_noderange_shortname_tags
nodels_noderange_table						nodels_noderange_table
nodels_noderange_tablecolumn				      <
nodels_noderange_table_equal					nodels_noderange_table_equal
nodels_noderange_table_unequal					nodels_noderange_table_unequal
nodels_noderange_table_uninclude				nodels_noderange_table_uninclude
							      >	nodels_noderange_tablecolumn
nodels_null							nodels_null
nodels_S						      <
nodels_table_include						nodels_table_include
nodels_tablevalue						nodels_tablevalue
nodels_tablevalue_tablecolumn					nodels_tablevalue_tablecolumn
nodels_v							nodels_v
noderange_10-20							noderange_10-20
							      >	noderange_XCAT_NODE_PREFIX
							      >	noderange_XCAT_NODE_SUFFIX
noderange_dir_tmp-nodelistfile					noderange_dir_tmp-nodelistfile
noderange_exclusion						noderange_exclusion
noderange_f1-f2_n1-n3						noderange_f1-f2_n1-n3
noderange_group1-group3						noderange_group1-group3
noderange_group_intersection					noderange_group_intersection
noderange_individual_grp					noderange_individual_grp
noderange_individual_node					noderange_individual_node
noderange_node_01-10					      <
noderange_node01_10					      <
noderange_node01-node10						noderange_node01-node10
							      >	noderange_node01_10
noderange_node10_plus_3						noderange_node10_plus_3
							      >	noderange_node_01-10
noderange_slash_urtestnode_dot_star				noderange_slash_urtestnode_dot_star
noderange_XCAT_NODE_PREFIX				      <
noderange_XCAT_NODE_SUFFIX				      <
noderm_err_node							noderm_err_node
noderm_h							noderm_h
noderm_noderange						noderm_noderange
noderm_null							noderm_null
							      >	nodestat_err_node
							      >	restorexCAT_h
restorexCATdb_a_p_V						restorexCATdb_a_p_V
restorexcatdb_p						      <
restorexCATdb_p_V						restorexCATdb_p_V
restorexCATdb_v							restorexCATdb_v
restorexCATdb_wrongpath						restorexCATdb_wrongpath
restorexCAT_h						      |	restorexcatdb_p
rmdef_dynamic_group						rmdef_dynamic_group
rmdef_group							rmdef_group
rmdef_node							rmdef_node
rmdef_null							rmdef_null
rmdef_template						      <
rmdef_t_err							rmdef_t_err
rmdef_t_node							rmdef_t_node
rmdef_t_o_network						rmdef_t_o_network
rmdef_t_o_node							rmdef_t_o_node
							      >	rmdef_template
run_command_with_XCATBYPASS					run_command_with_XCATBYPASS
tabdump_d							tabdump_d
tabdump_d_nodehm						tabdump_d_nodehm
tabdump_f_d							tabdump_f_d
tabdump_h							tabdump_h
tabdump_help							tabdump_help
tabdump_null							tabdump_null
tabdump_table							tabdump_table
tabdump_v							tabdump_v
tabdump_w_equal							tabdump_w_equal
tabdump_w_ge							tabdump_w_ge
tabdump_w_gt							tabdump_w_gt
tabdump_w_le							tabdump_w_le
tabdump_w_lt							tabdump_w_lt
tabdump_w_match							tabdump_w_match
tabdump_w_ne							tabdump_w_ne
tabdump_w_notmatch						tabdump_w_notmatch
tabgrep_err							tabgrep_err
tabgrep_h							tabgrep_h
tabgrep_node							tabgrep_node
tabgrep_null							tabgrep_null
							      >	tabprune_V_a_eventlog
							      >	tabprune_V_n_auditlog
tabprune_a_eventlog						tabprune_a_eventlog
tabprune_h							tabprune_h
tabprune_i_auditlog						tabprune_i_auditlog
tabprune_p_auditlog						tabprune_p_auditlog
tabprune_v							tabprune_v
tabprune_V_a_eventlog					      <
tabprune_V_n_auditlog					      <
tabrestore_err							tabrestore_err
tabrestore_h							tabrestore_h
tabrestore_null							tabrestore_null
tabrestore_table						tabrestore_table
xcatd_start							xcatd_start
xcatd_stop							xcatd_stop
xcatsnap_b_d							xcatsnap_b_d
xcatsnap_h							xcatsnap_h
xcatsnap_null							xcatsnap_null
xcatstanzafile_attribute					xcatstanzafile_attribute
xcatstanzafile_colon						xcatstanzafile_colon
xcatstanzafile_objtype						xcatstanzafile_objtype
xcatstanzafile_specificvalue					xcatstanzafile_specificvalue
```

File: xCAT-test/autotest/testcase/makentp/cases0 [hunk 1]
```diff
@@ -1,18 +1,21 @@
 start:makentp_v
+
 descriptiop:makentp -v
```
Comment (hu-weihua) [inline xCAT-test/autotest/testcase/makentp/cases0]: why are here need a blank line?

Comment (hu-weihua) [reply xCAT-test/autotest/testcase/makentp/cases0]: There are a lot of blanks too.

Comment (hu-weihua) [reply xCAT-test/autotest/testcase/makentp/cases0]: There are a lot of blanks below too.

File: xCAT-test/autotest/testcase/mkvm/cases0 [hunk 2]
```diff
@@ -1,4 +1,5 @@
 start:mkvm_i_l
+label:others,hctrl_hmc
```
Comment (hu-weihua) [inline xCAT-test/autotest/testcase/mkvm/cases0]: why does the case ``mkvm_i_l`` belong to ``hctrl_hmc``?

File: xCAT-test/autotest/testcase/ngpfb/cases0 [hunk 3]
```diff
@@ -1,7 +1,9 @@
 #comments:make sure you have compute node definition on your MN
 #this case is just supplement for some commands,after your executing,check output from log
 start:lsslp_cmm
+
```
Comment (hu-weihua) [inline xCAT-test/autotest/testcase/ngpfb/cases0]: It is better to remove all of such useless blank lines.

File: xCAT-test/autotest/testcase/rmvm/cases0 [hunk 4]
```diff
@@ -1,4 +1,5 @@
 start:rmvm_noderange
+label:others,hctrl_hmc
```
Comment (hu-weihua) [inline xCAT-test/autotest/testcase/rmvm/cases0]: Does this case belong to ``hctrl_hmc``. There is a label ``virtualization`` which I think it is better.

**Comment by hu-weihua** [comment]
@tingtli, could you run ``XCATTEST_CASEDIR="/xcat-core/xCAT-test/autotest/testcase/"  xcattest -l -s "ci_test"`` under your repo?  I thought there was format error in your PR, the result of above command did not make sense.

**Comment by hu-weihua** [comment]
@tingtli , Every case must have a ``Vertical Dimension Labels``, otherwise it will trigger mistake.

For example:
```
makedhcp_n_linux
```

**Comment by hu-weihua** [comment]
issue https://github.com/xcat2/xcat-core/issues/5773 may block this PR

**Comment by tingtli** [comment]
@hu-weihua 
here is the output
```
[root@c910f02c02p15 autotest]# XCATTEST_CASEDIR="/xcat-core/xCAT-test/autotest/testcase/" xcattest -l -s "ci_test"
dumpxCATdb_a_p_nullskiptables_V
dumpxCATdb_p_skiptables
mkdef_f
nodels_err_symbol
nodech_noderanage_table_at
tabdump_w_gt
chtab_err_table
chdef_null
restorexCAT_h
nodech_noderange_table_comma_arrow
makehosts_h
lsdef_t_i_o
nodels_S
lsdef_template_switch_template
noderange_individual_grp
tabdump_help
lsdef_a
nodech_noderange_table_unequal
chdef_t_network
noderange_node01_10
nodech_noderange_table_equal
noderange_dir_tmp-nodelistfile
nodegrpch_h
nodech_noderange_shortname_tags
mkdef_template_cec_template_without_remainder
nodeadd_noderange_nodetype
tabrestore_null
chdef_p
noderange_10-20
lsxcatd_a
noderm_h
makehosts_n_noderange
tabdump_w_lt
tabrestore_err
tabgrep_null
lsdef_t_eventlog
check_mn_cluster_log_file
tabrestore_table
lsdef_t_w
nodech_h
xcatsnap_b_d
chdef_z
tabgrep_err
lsdef_s
nodeadd_noderange
lsdef_nics
tabprune_V_n_auditlog
mkdef_node
nodech_v
noderm_err_node
nodels_table_include
chtab_v
rmdef_t_err
nodeadd_err_symbol
nodels_noderange_table_unequal
mkdef_z
tabprune_v
noderange_group1-group3
tabdump_table
dumpxCATdb_p_nullskiptables
chdef_t_node
nodels_h
assign_certain_command_permission
nodels_null
nodels_noderange
tabdump_f_d
nodels_v
nodech_noderange_shortname_groups
lsdef_t_site
lsdef_t_h_i
nodels_tablevalue
tabdump_w_match
makedns_h
noderange_slash_urtestnode_dot_star
noderm_noderange
lsxcatd_h
tabgrep_node
rmdef_node
nodegrpch_err
lsdef_t_auditlog
lsxcatd_d
chtab_d
rmdef_group
mkdef_multiple_keys
chtab_err_symble
gettab_err
makehosts_help
rmdef_t_o_node
tabdump_d_nodehm
restorexCATdb_wrongpath
mkdef_null
nodegrpch_groups
mkdef_regex_nicsip
xcatd_start
nodels_noderange_table
lsdef_template
tabdump_w_ne
tabprune_a_eventlog
tabrestore_h
noderange_exclusion
lsdef_template_with_invalid_name
makedhcp_h
nodels_b
lsdef_t_o_l_z
nodels_noderange_table_equal
dumpxCATdb_p_V
chdef_multiple_keys
restorexCATdb_p_V
xcatstanzafile_attribute
noderange_XCAT_NODE_PREFIX
noderange_node_01-10
nodech_noderange_table_arrow
mkdef_template_invalid_template
tabdump_w_ge
nodeadd_h
rmdef_t_node
nodech_noderange_table_comma
lsdef_t
xcatsnap_h
nodech_error_node
rmdef_template
noderange_node01-node10
gettab_key_table
run_command_with_XCATBYPASS
mkdef_node_with_a_node_template
nodeadd_null
mkdef_dynamic_group
nodels_tablevalue_tablecolumn
nodels_H
dumpxCATdb_a_p_nullskiptables
restorexCATdb_v
chdef_group
chtab_modify_node
nodech_noderange_table
tabprune_i_auditlog
chtab_modify_key
dumpxCATdb_p_nullskiptables_V
lsxcatd_null
makehosts_n
mkdef_t_network
nodels_noderange_shortname_groups
xcatstanzafile_objtype
noderange_node10_plus_3
chtab_h
dumpxCATdb_a_p_skiptables
tabprune_V_a_eventlog
nodels_noderange_table_uninclude
tabdump_v
noderange_group_intersection
tabdump_h
rmdef_null
chtab_null
nodech_noderange_shortname_mgt
nodech_error_table
lsdef_t_o_l
noderm_null
nodegrpch_v
nodech_noderange_table_uninclude
tabdump_d
tabprune_h
rmdef_dynamic_group
mkdef_github_issue2582
chdef_n
tabdump_w_equal
makedhcp_n
chdef_m
chdef_group_p
mkdef_group
chdef_t_o_error
tabdump_w_le
xcatd_stop
xcatsnap_null
gettab_h
nodels_noderange_shortname_mgt
nodels_err_noderange
nodech_noderange_table_include
tabdump_w_notmatch
lsdef_t_policy
mkdef_template_switch_template_without_attribute
nodels_noderange_shortname_tags
restorexCATdb_a_p_V
lsdef_null
tabprune_p_auditlog
nodech_delete
rmdef_t_o_network
xcatstanzafile_colon
noderange_XCAT_NODE_SUFFIX
noderange_f1-f2_n1-n3
chdef_dynamic_group
noderange_individual_node
gettab_H
makedhcp_help
dumpxCATdb_v
tabdump_null
makehosts_null
lsdef_t_err
mkdef_t_o_error
xcatstanzafile_specificvalue
disable_root_permission_in_policy_table
nodels_noderange_tablecolumn
dumpxCATdb_h
chdef_template
nodeadd_v
restorexcatdb_p
tabgrep_h
makenetworks_help
makeknownhosts_h
makeconservercf_noderange
makeconservercf_null
makenetworks_h
makedhcp_n_linux
makenetworks_v
makeconservercf_d
makenetworks_d
```

**Comment by hu-weihua** [comment]
It is ok for me, thanks @tingtli

**Pull request merged**